0-bootstrap/modules/groups/outputs.tf

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "created_required_groups" {
+  description = "List of the Required Groups created by the module."
+  value       = var.create_groups_holder.required_groups
+
+  //This depends_on is necessary otherwhise the entered Required groups would be returned before they were properly created
+  depends_on = [module.required_group]
```

Comment:
Can we use `id` op from module.required_group which is a computed attrib? That should help us get rid of this.

Reply:
Done.